Makefile.kokkos

```diff
@@ -1014,86 +1029,123 @@ ifeq ($(KOKKOS_INTERNAL_USE_OPENMPTARGET), 1)
   endif
 endif
 
+ifeq ($(KOKKOS_INTERNAL_USE_OPENACC), 1)
+  ifeq ($(KOKKOS_INTERNAL_COMPILER_CLANG), 1)
+      KOKKOS_INTERNAL_CUDA_ARCH_FLAG=-fopenmp-targets=nvptx64-nvidia-cuda -Xopenmp-target=nvptx64-nvidia-cuda -march
+      KOKKOS_INTERNAL_AMD_ARCH_FLAG=-fopenmp-targets=amdgcn-amd-amdhsa -Xopenmp-target=amdgcn-amd-amdhsa -march
```

Comment:
So `OpenACC` using `openmp-target` flags?

Reply:
The Clacc compiler is built on top of the LLVM OpenMP compiler, and thus it internally uses the existing LLVM OpenMP flags.